crates/topos-sequencer-subnet-runtime/tests/subnet_contract.rs

```diff
@@ -918,3 +979,246 @@ async fn test_subnet_send_token_processing(
     context.shutdown().await?;
     Ok(())
 }
+
+/// Test multiple send token events in a block
+/// Test is slow, block time is 12 seconds
+#[rstest]
+#[test(tokio::test)]
+#[serial]
+async fn test_subnet_multiple_send_token_in_a_block(
```

Comment:
What do you think about adding `#[timeout]` here, i.e. https://github.com/la10736/rstest#test-timeout

Reply:
It has to be some very big failsafe timeout, because it is hard to assume how long the test will last - it has many steps with long per block time

Reply:
Right, I already put 10 minutes